## 4) Calculate probabilities or identify scores that are very far from the mean using dnorm: 

After the test, Mr Gittu finds out that the actual mean was 68% and s.d. of 11%, not bad! He's using dnorm to figure out how much of an outlier certain scores are. If he wants to know how unusual a score of 50% is:

In [3]:
from pystats.dnorm import dnorm
result = dnorm(50, mean = 68, sd = 11)
print(result)

    x       PDF
0  50  0.009507


This output shows the probability density function (PDF) value of a normal distribution for a score of 50, given a mean of 68 and a standard deviation of 11. The result indicates that the PDF value at x=50 is approximately 0.009507. This confirms that 50 is on the tail end of the normal distribution curve, which is why it has a low probability density.